app/views/schools/user_tariffs/_meters.html.erb

```diff
@@ -13,5 +13,5 @@
     </div>
   </div>
 <% else %>
-  <p>There are no meters associated with this tariff</p>
+  <p><%= t('schools.user_tariffs.meters.there_are_no_meters_associated_with_this_tariff') %></p>
```

Comment:
Minor comment, and I've not noticed if this is a convention or not, but this label is rather long